qlib/data/data.py

```diff
@@ -122,6 +122,16 @@ def _get_calendar(self, freq, future):
             H["c"][flag] = _calendar, _calendar_index
         return _calendar, _calendar_index
 
+    def get_calendar_day(self, freq="day", future=False):
```

Comment:
`get_calendar_day` just for high-freq, if there are other ways to implement it

Reply:
We don't need to put this in `qlib/data/data.py`